sdk/web-pubsub/web-pubsub-client/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationDir": "./types",
+    "module": "es6",
+    "moduleResolution": "node",
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "outDir": "./dist-esm",
+    "rootDir": ".",
+    "target": "es6",
```

Comment:
certainly you can target newer than es6, probably ES2017 like we use today is fine, but you could go as far as ES2020: https://github.com/Azure/azure-sdk-for-js/issues/23641

Reply:
probably not worth overriding so many things though?